vita3k/gui/src/game_selector.cpp

```diff
@@ -171,9 +205,9 @@ void draw_game_selector(GuiState &gui, HostState &host) {
                 ImGui::Image(gui.game_selector.icons[game.title_id], ImVec2(icon_size, icon_size));
                 if (ImGui::IsItemHovered()) {
                     if (host.cfg.show_game_background) {
-                        if (gui.game_backgrounds.find(game.title_id) == gui.game_backgrounds.end())
+                        if (!gui.game_backgrounds[game.title_id])
                             load_game_background(gui, host, game.title_id);
-                        else
+                        else if (gui.current_background != gui.game_backgrounds[game.title_id])
```

Comment:
nuh uh

Reply:
Yes for me i no want set spam image if already present